### PR TITLE
Remove experimental badge

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

**Comment by loosebazooka** [comment]
@meltsufin

File: README.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-[![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)
```
Comment (patflynn) [inline README.md]: don't you want to replace this with the 'stable' badge?

Comment (loosebazooka) [inline README.md]: ptal

**Comment by meltsufin** [comment]
Do you want to remove the "(BETA)" from the title too?

**Comment by loosebazooka** [comment]
yes

**Pull request merged**